R/ResampleDesc.R

```diff
@@ -64,6 +64,9 @@
 #'     else it will be a fraction of the total training indices. IE for 100 training sets and a value of .2, the increment
 #'     of the resampling indices will be 20. Default is \dQuote{horizon} which gives mutually exclusive chunks
 #'      of test indices.}
+#'   \item{fixed (`logical(1)`)}{Whether indices supplied via argument 'blocking' in the task should be used in resampling. Default is `FALSE`.
+#'     'grouping' only works with 'CV' and the supplied indices must match the number of observations.}
+#'   \item{blocking.cv (`logical(1)`)}{Should 'blocking' be used in 'CV'? Default to `FALSE`}
```

Comment:
It sounds like this does the same thing as `fixed`.

Reply:
I added more detail to the doc and a link to the (not yet existing) tutorial page.